drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -709,19 +709,69 @@
 /*** ACTIONS TOOLBAR ***/
 
 .xblock--drag-and-drop .actions-toolbar {
-    min-height: 3.75em;
-    width: auto;
     position: relative;
+    background-color: #1d5280;
+    margin-top: 0;
+    height: 62px;
+    padding: 9px 20px;
+    box-sizing: border-box;
+
 }
 
 .xblock--drag-and-drop .actions-toolbar .action-toolbar-item {
     display: inline-block;
     margin: 10px 0;
 }
 
+.xblock--drag-and-drop .actions-toolbar .submit-answer-button{
+    padding: 0.625em 2.45em;
```

Comment:
It seems the padding breaks the mobile view. Could you reduce it?

![screen shot 2018-08-30 at 18 37 09](https://user-images.githubusercontent.com/5691347/44931838-5f651000-ad39-11e8-98ff-9399b5a619ec.png)